HerPortal.ManagementShell/Program.cs

```diff
@@ -0,0 +1,228 @@
+﻿using HerPortal.BusinessLogic.Models;
+using HerPortal.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace HerPortal.ManagementShell
+{
+    public static class Program
+    {
+        public enum UserStatus
+        {
+            New,
+            Active
+        }
+
+        public static void Main(string[] args)
+        {
+            var contextOptions = new DbContextOptionsBuilder<HerDbContext>()
+                .UseNpgsql(
+                    @"UserId=postgres;Password=postgres;Server=localhost;Port=5432;Database=herportaldev;Integrated Security=true;Include Error Detail=true;Pooling=true")
```

Comment:
This feels like something that shouldn't be committed (even if it's a default password😅). Is this just intended for local development and we'll have a different setup elsewhere?

Reply:
Ah yes definitely! This was in the back of my mind to do, but forgot to remove from the PR. I'll double check about environment variables to replace here :)